packages/components/ng-ovh-payment-method/src/components/register/sepa-information-modal/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ovh_payment_method_register_sepa_information_modal_title": "Comptes bancaires zone SEPA",
+  "ovh_payment_method_register_sepa_information_modal_confirm": "J'ai compris",
+  "ovh_payment_method_register_sepa_information_modal_part_1": "Pour mettre en place un prélèvement bancaire, vous devez être en possession d’un compte bancaire détenu dans un pays de l’espace unique de paiements en euros, ",
+  "ovh_payment_method_register_sepa_information_modal_part_1_emphasis": "la zone SEPA.",
```

Comment:
Are sure translation could be managed for all languages by splitting one sentence in two parts (especially because of grammar rules) ?
Maybe using `<strong></strong>` would be safer.